pyvista/plotting/composite_mapper.py

```diff
@@ -0,0 +1,1032 @@
+        """Get or set the visibility of a block.
+
+        Examples
+        --------
+        Hide the first block of a composite dataset.
+
+        Note how the zero index is the entire multiblock, so we have to add 1
+        to our indexing to access the right block.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.block_attr[1].visible = False
+        >>> pl.show()
+
+        """
+        if not self._has_visibility:
+            return None
+        return self._attr.GetBlockVisibility(self._block)
+
+    @visible.setter
+    def visible(self, new_visible: bool):
+        if new_visible is None:
+            self._attr.RemoveBlockVisibility(self._block)
+            self._attr.Modified()
+            return
+        self._attr.SetBlockVisibility(self._block, new_visible)
+
+    @property
+    def opacity(self) -> Optional[float]:
+        """Get or set the opacity of a block.
+
+        If opacity has not been set this will be ``None``.
+
+        Warnings
+        --------
+        VTK 9.0.3 has a bug where changing the opacity to less than 1.0 also
+        changes the edge visibility on the block that is partially transparent.
+
+        Examples
+        --------
+        Change the opacity of the second block of the dataset.
+
+        Note how the zero index is the entire multiblock, so we have to add 1
+        to our indexing to access the right block.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.block_attr[2].opacity = 0.5
+        >>> pl.show()
+
+        """
+        if not self._has_opacity:
+            return None
+        return self._attr.GetBlockOpacity(self._block)
+
+    @opacity.setter
+    def opacity(self, new_opacity: float):
+        """Get or set the visibility of a block."""
+        if new_opacity is None:
+            self._attr.RemoveBlockOpacity(self._block)
+            self._attr.Modified()
+            return
+
+        self._attr.SetBlockOpacity(self._block, new_opacity)
+
+    @property
+    def pickable(self) -> Optional[bool]:
+        """Get or set the pickability of a block.
+
+        Examples
+        --------
+        Make the cube of a multiblock dataset pickable and the sphere unpickable.
+
+        Note how the zero index is the entire multiblock, so we have to add 1
+        to our indexing to access the right block.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.block_attr[1].pickable = True
+        >>> mapper.block_attr[2].pickable = False
```

Comment:
This is also a blank-looking figure, but the feature being exemplified here is not visible anyway. Should we perhaps also enable picking in the example with some simple callback (random color? shifting position?), so that interactively running the example as-is will be instructive? In that case blankness would be fine too.

Or perhaps just link to `enable_block_picking()`.

Reply:
Linked to `enable_block_picking()` in 8835c292f03a212cc953200b58f421e82e67c120.